lambdas/process-journey-event/src/main/java/uk/gov/di/ipv/core/processjourneyevent/statemachine/NestedJourneyTypes.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.di.ipv.core.processjourneyevent.statemachine;
+
+import lombok.Getter;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+
+@Getter
+@ExcludeFromGeneratedCoverageReport
+public enum NestedJourneyTypes {
+    // When a new nested-journey is added, it needs to be added here
+    // as the statemachine will look here when initialising
```

Comment:
Can we add a comment referencing `constants.mjs` as well - to indicate that you need to update both?